uni/ulsp/file_handler.icn

```diff
@@ -486,6 +488,140 @@ class FileHandler(
       return global_variables
    end
 
+# -------------------------------------------------------------------------------------------------
+# ---------------- Ranges (get, set) --------------------------------------------------------------
+# -------------------------------------------------------------------------------------------------
+
+   method setInternalPositions()
+      local lineNum, _line, procedureName, startPos, endPos, className, methodName, variableName,
+            open_blocks, ending, currentClass
+
+      internal_positions := [
+         "classes" : [];
+         "procedures" : [];
+         "methods" : [];
+         "variables" : []
```

Comment:
These will end up being lists now, not tables since `[ ]` is a list initializer syntax.  To force it to be a table you can do  `[:]` which is short of `[&null: &null]` though it is common to just use `table()` as you did before.